src/readline/Makefile.am

```diff
@@ -6,4 +6,5 @@ libreadline_la_SOURCES = readline.c history.c funmap.c keymaps.c
 noinst_HEADERS = chardefs.h emacs_keymap.h history.h keymaps.h \
 	readline.h vi_keymap.h
 
-
+install-exec-hook:
+	$(LN_S) $(DESTDIR)$(libdir)/libreadline.so $(DESTDIR)$(libdir)/libreadline0.so
```

Comment:
The symbolic link syntax seems reversed. And why is it necessary anyway. Typically, libtool creates the links.